### PR TITLE
Rename-It! Shell Extension on x64 Systems

### PR DESCRIPTION
**Pull request by jpix**
Fix x64 context menu shell extension problem. Update project to Visual
Studio 2015

### CODE REVIEW

**Comment by wernight** [comment]
Nice work. I forgot how verbose Visual Studio can be for source control.

**Comment by wernight** [comment]
I don't see updates to the setup. Am I missing something?

**Comment by jpix** [comment]
Nope, you're right once again. Thanks.

 

All requested changes have been done

 

•             Resource strings reverted

•             Languages restored

•             Setups updated

 

 

Da: Werner Beroux [mailto:notifications@github.com] 
Inviato: mercoledì 17 febbraio 2016 10.53
A: wernight/renameit <renameit@noreply.github.com>
Cc: jpix <gppischedda@gmail.com>
Oggetto: Re: [renameit] Rename-It! Shell Extension on x64 Systems (#39)

 

I don't see updates to the setup. Am I missing something?

File: apps/Rename-It!/RenameIt.rc [hunk 1]
```diff
@@ -1024,12 +1058,12 @@ END
 
 IDD_WIZ_APPEND DIALOGEX 0, 0, 186, 90
 STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Append Name"
+CAPTION "Aggiungi Nome"
```
Comment (wernight) [inline apps/Rename-It!/RenameIt.rc]: Missed extra translation?

File: apps/Rename-It!/src/Gui/AboutDlg.cpp [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #include "StdAfx.h"
 #include "AboutDlg.h"
 
+//#define IDC_WEB_SO_ON_DEM 1133
```
Comment (wernight) [inline apps/Rename-It!/src/Gui/AboutDlg.cpp]: Should revert those comments (you can keep them on another branch if you don't want to lose them locally).

File: apps/SimpleExt/SimpleExt.rc [hunk 3]
```diff
@@ -93,20 +90,20 @@ VS_VERSION_INFO VERSIONINFO
...
-        BLOCK "040c04b0"
+        BLOCK "000904b0"
         BEGIN
-            VALUE "FileDescription", "Rename-It! Shell Extension"
-            VALUE "FileVersion", "2, 0, 0, 1"
+            VALUE "FileDescription", "Rename-It! v4.x Shell Extension for x64"
+            VALUE "FileVersion", "4.1.1.1"
             VALUE "InternalName", "SimpleEx"
-            VALUE "LegalCopyright", "Copyright (C) 2003-2005"
+            VALUE "LegalCopyright", "Under Apache License"
```
Comment (wernight) [inline apps/SimpleExt/SimpleExt.rc]: It's still "Copyright (c) 2003-2016 Werner Beroux"

*Adding* that it's under Apache License in the UI like a good follow up improvement, with a link to GitHub.

**Comment by wernight** [comment]
Before it works a follow up PR or in same PR need to update https://github.com/wernight/renameit/blob/master/setup/NSIS.nsi#L116 to include the x64 and install it based on the local system I believe.

**Comment by wernight** [comment]
Thanks for the commit it seems fine.

**Pull request merged**